controllers/persoon.ts

```diff
@@ -0,0 +1,62 @@
+import { bestuursperiode } from '../data-access/bestuursperiode';
+import { person } from '../data-access/persoon';
+import { STATUS_CODE } from '../util/constants';
+import { HttpError } from '../util/http-error';
+
+export const personUsecase = {
+  findOnfhankelijkeFractieUri,
+  updateCurrentFractie,
+};
+
+async function findOnfhankelijkeFractieUri(
+  personId: string,
+): Promise<string | null> {
+  const isPerson = await person.exists(personId);
+
+  if (!isPerson) {
+    throw new HttpError(
+      `No person found for given id: ${personId}`,
+      STATUS_CODE.BAD_REQUEST,
+    );
+  }
+
+  const onafhankelijkerFractieUri =
+    await person.findOnafhankelijkeFractieUri(personId);
+
+  return onafhankelijkerFractieUri !== null ? onafhankelijkerFractieUri : null;
+}
+
+async function updateCurrentFractie(
```

Comment:
I think it makes more sense to do this on the mandataris. From the mandataris you can easily get to the persoon and bestuursperiode, which would simplify this whole logic I think.